invenio_i18n/babel.py

```diff
@@ -53,16 +54,18 @@ def __init__(self, paths=None, entry_point_group=None, domain="messages"):
         :param domain: Name of message catalog domain.
             (Default: ``'messages'``)
         """
-        self.paths = []
+        super().__init__(domain=domain)
```

Comment:
Have you been able to test these changes?

Reply:
I only tested with the tests from invenio-i18n. My plan has been to show you this PR and then go throw every necessary package and do the changes and test it out and then do a merge the PR.